README.md

```diff
@@ -162,12 +161,13 @@ Download and install one of the following WebDriver (depending on the browser ty
 | :---:                                                                                                | :---                                                                          | :---                                                                                                             |
 | ![Chrome](https://raw.githubusercontent.com/alrra/browser-logos/main/src/chrome/chrome_16x16.png)    | [Chrome](https://sites.google.com/chromium.org/driver/downloads)              | [Status](https://chromium.googlesource.com/chromium/src/+/master/docs/chromedriver_status.md)                    |
 | ![Edge](https://raw.githubusercontent.com/alrra/browser-logos/main/src/edge/edge_16x16.png)          | [Edge](https://developer.microsoft.com/en-us/microsoft-edge/tools/webdriver/) | [Status](https://chromium.googlesource.com/chromium/src/+/master/docs/chromedriver_status.md) [^1] |
-| ![Firefox](https://raw.githubusercontent.com/alrra/browser-logos/main/src/firefox/firefox_16x16.png) | [Firefox](https://github.com/mozilla/geckodriver/releases/latest)             | [Status](https://developer.mozilla.org/en-US/docs/Web/WebDriver#specifications)                                  |
+| ![Firefox](https://raw.githubusercontent.com/alrra/browser-logos/main/src/firefox/firefox_16x16.png) | [Firefox](https://github.com/mozilla/geckodriver/releases/latest)             | [Status](https://developer.mozilla.org/en-US/docs/Mozilla/QA/Marionette/WebDriver/status) [^2}                                  |
 
-Limitation notice. [^2]
+Limitation notice. [^3]
 
 [^1]: "Microsoft Edge" (is chromium based) which means it's a WebDriver implementation derived from ChromeDriver. The status or limitations are at least the same as for ChromeDriver.
```

Comment:
@mlipok I'm don't know if this is correct. What "proof" do you have that `msedgedriver` is derived from `chromedriver`?

Reply:
Where this statement was used first time.

Reply:
Good question. I assumed that you had added this, but maybe it was @Sven-Seyfert. 🤔

Reply:
Sorry for the confusion. It looks like it was added by @Sven-Seyfert in 0d9a23aa0bd650e92ea16cbd42fa93bcd489eca5. So what do you think about this @Sven-Seyfert ?

Reply:
> Sorry for the confusion. It looks like it was added by @Sven-Seyfert

That's what I thought

Reply:
Sorry folks for the late response regarding this. Maybe I misinterpret this:

![grafik](https://user-images.githubusercontent.com/29656921/155129475-4b801f36-8a16-4ecb-b630-681b66799672.png)

> [^2]: Login Required

The URL change lead to the login page. That's why I thought it's better to reference the chromium state starting from assumption above.

Reply:
> Maybe I misinterpret this

I know they are both based on Chromium browser, but I think they maintain separate codebases for their customizations. 

> The URL change lead to the login page which isn't pretty handy. 

Agreed. However, they do offer a login via Github and I felt this was better than referring to the webdriver specs.

Reply:
> Dan: I know they are both based on Chromium browser, but I think they maintain separate codebases for their customizations.

I believe you are right, fair enough. Thanks for being so aware of these little things 👍 .

> Dan: Agreed. However, they do offer a login via Github and I felt this was better than referring to the webdriver specs.

Okay, acceptable and understandable. Good job 😁 .